package.json

```diff
@@ -21,7 +21,7 @@
     "lint:fix": "yarn lint --fix",
     "prettier": "prettier --list-different \"**/*.{ts,tsx}\"",
     "prettier:fix": "prettier --write \"**/*.{ts,tsx}\"",
-    "precommit": "lint-staged",
+    "precommit": "gulp stats && lint-staged",
```

Comment:
We build stats on every commit, so that component count and size is updated with each change to the src.  This is quite slow, so optimizations to the speed are likely warranted.